src/Templates/SingleProductTemplateCompatibility.php

```diff
@@ -450,4 +448,27 @@ function( $carry, $block ) {
 			array()
 		);
 	}
+
+	/**
+	 * Inject the hooks after the div wrapper.
+	 *
+	 * @param string $block_content Block Content.
+	 * @param array  $hooks Hooks to inject.
+	 * @return array
+	 */
+	private function inject_hooks_after_the_wrapper( $block_content, $hooks ) {
+		$closing_tag_position = strpos( $block_content, '>' );
```

Comment:
I was just curious, Will there be always `>`? Do we always want to insert after first `>` character? 🤷🏻‍♂️

Reply:
> I was just curious, Will there be always >? 

Yes, this class adds this div:

```
<div class="wp-block-group woocommerce block ...">
```

> Do we always want to insert after first > character? 🤷🏻‍♂️

Yes! We want to inject the hooks after the close of the div tag. 

Does it make sense?

Reply:
That makes sense. Thanks for the explanation @gigitux. I just wanted to make sure that there will always be `>` character. 🤝